### ISSUE TITLE
Find way to teaserify feed items

### ISSUE THREAD

**Issue by giorgiosironi** [hivereview/thehive#266]
In https://github.com/hivereview/thehive/issues/261 we are displaying the full content (but collapsible) rather than a `See more` pattern. 

- [x] should work in a screen reader
- [x] should be keyboard navigable
- [x] ~shouldn't duplicate content in the DOM~
- [x] server should not be required to split content into teaser and rest-of-the-content
- [x] should display everything if there's no JavaScript on the client side
- [x] does not have to support page fragment link targets within collapsed content
- [x] should not rule out opening all/one collapsible sections when the page loads
- [x] build toggle in JavaScript
- [x] script in static js file
- [x] styles in the css file
- [x] renaming 
- [x] use data attribute for state
- [x] cursor should be a finger

Screenreader / keyboard flow order of an item in the review list should be:
1. date
1. event title
1. event teaser text
1. 'read more' (or whatever) toggle
  - when currently toggled closed:
    ~- if the user activates the toggle control to open, the section expands and narration continues with the flow of the text in that review~ when section expanded, reading begins from the start of the item details, this is consistent with the behaviour with `<details>`/`summary`.
     - if the user does not activate the toggle control to open, the narration moves to the next item in the review list
  - when currently toggled open:
    - hide toggle from a11y tree (will remain open for screen readers)
    - keep toggle keyboard navigable

**Comment by davidcmoulton** [comment]
This could be done by js removing all of the review details bar the teaser text from the DOM, and reintroducing it when the accordian is opened. Initial tests screen reader investigations indicate this might be good enough.

**Comment by BlueReZZ** [comment]
Sounds like we could reuse https://github.com/elifesciences/pattern-library/blob/d64d27b8bfef4c1ac796bac55e4cc36344f23e0d/assets/js/components/ToggleableCaption.js by making our HTML look like https://github.com/elifesciences/pattern-library/blob/d64d27b8bfef4c1ac796bac55e4cc36344f23e0d/test/toggleablecaption.html and take it from there as it appears to do what @davidcmoulton describes above.
